src/compas_rhino/geometry/curves/__init__.py

```diff
@@ -1,19 +1,21 @@
-from .curve import RhinoCurve
+from .curve import RhinoCurve  # noqa: F401
 from .nurbs import RhinoNurbsCurve
 
+from compas.geometry import Curve
+from compas.geometry import NurbsCurve
 from compas.plugins import plugin
 
 
 @plugin(category="factories", requires=["Rhino"])
 def new_curve(cls, *args, **kwargs):
-    curve = object.__new__(RhinoCurve)
+    curve = super(Curve, cls).__new__(cls)
```

Comment:
in my opinion this handles the inheritance chain better...

Reply:
when I do this in `Brep` I get an instance of `compas.geometry.Brep` even in Rhino, which is unexpected in my opinion.

Reply:
i think it would be better if it returns `compas.geometry.NurbsCurve` and not the Rhino version, because then the returned object matches the object you tried to create (and its documentation), but this requires that the Rhino implementation completely aligns with the pluggable API. and since that is currently not the case, it would create unexpected results.

was also included more or less by accident so we can ignore the issue for now...

Reply:
If it returns a `NurbsCurve` then every single method has to be a pluggable/plugin, that would be really cumbersome, no?